integration-tests/src/test/java/oracle/kubernetes/operator/ItOperatorUpgrade.java

```diff
@@ -238,41 +141,74 @@ public void testOperatorUpgradeFrom2_3_0() throws Exception {
    * @param restart boolean parameter used to determine if a restart of domain is checked
    * @throws Exception when upgrade fails or basic usecase testing or scaling fails.
    */
-  private void upgradeOperator(boolean restart) throws Exception {
+  private void upgradeOperator() throws Exception {
     operator.callHelmUpgrade("image=" + OP_TARGET_RELEASE);
-    if (restart) {
-      checkDomainRollingRestarted();
-    }
-    checkOperatorVersion();
+    checkDomainNotRestarted();
+    checkCrdVersion();
+    // checkDomainApiVersion(); Not Needed 
     testClusterScaling(operator, domain, false);
   }
 
   /**
-   * checks the expected version of the upgraded operator in a loop. In Jenkins it takes nearly 8
-   * minutes to show the updated value of the domain CRD.
+   * Checks Expected/Upgraded ApiVersion of the Domain Object in a loop. 
+   * In Jenkins it takes nearly 8 minutes to show the updated value 
+   * 
+   * The .apiVersion of the domain will not be updated until the domain is 
+   * written again (e.g. by the operator when updating status)
+   * This contributes to long time period. We can skip this check 
    *
    * @throws Exception when version does not match
    */
-  private void checkOperatorVersion() throws Exception {
+  private void checkDomainApiVersion() throws Exception {
```

Comment:
remove this method as its not used

Reply:
Keeping it if we can decrease the time by revisiting the product code in future.

Reply:
@rjeberhard confirmed we don't need to verify domain resource apiVersion, checking crd apiVersion is enough. Ryan, can you please confirm again.

Reply:
Yes, correct.  We do not need to confirm that the operator updates the .apiVersion of a Domain resource.  I'm just interested in the CRD version and then the checks that the pods don't roll and that the domain remains manageable.

Reply:
@anpanigr you can remove the method

Reply:
removed the method to check DomainApi version